js/runner_test.go

```diff
@@ -1819,7 +1819,9 @@ func TestVUPanic(t *testing.T) {
 			require.Len(t, entries, 1)
 			assert.Equal(t, logrus.ErrorLevel, entries[0].Level)
 			require.True(t, strings.HasPrefix(entries[0].Message, "panic: here we panic"))
-			require.True(t, strings.HasSuffix(entries[0].Message, "Goja stack:\nfile:///script.js:3:4(12)"))
+			// broken since goja@f3cfc97811c0b4d8337902c3e42fb2371ba1d524 see
+			// https://github.com/dop251/goja/issues/179#issuecomment-783572020
+			// require.True(t, strings.HasSuffix(entries[0].Message, "Goja stack:\nfile:///script.js:3:4(12)"))
```

Comment:
Are we OK with this? Is this a breaking change then? Should we test with the re-throwing workaround suggested in that comment?

Reply:
I made an issue about discussing that https://github.com/loadimpact/k6/issues/1906. 

As I mention there I don't think that we have ever actually used this functionality really and the "workaround" is both error prone IMO and will need a lot of work. 

Regardless I am of the opinion that we can have it not work for some time and either fix it a later PR or just leave it until we actually need it again, and figure out what to do then.

Reply:
OK. Hopefully not a lot of users were relying on this, but like you mentioned, it wasn't very useful to begin with.

Reply:
At least the current behaviour was *not* helpful as combined with the lack of anything but the last frame and the  fact that even that line number is usually wrong I spent an hour or two debugging the wrong place before deciding to ignore the "goja stacktrace" and just go with logic and the golang stacktrace